src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -380,6 +343,82 @@ else if (callVariantContextType == INDEL) {
         return 0;
     }
 
+    public static boolean classifyVariants(final Optional<VariantContext> truthContext,
+                                           final String truthSample,
+                                           final Optional<VariantContext> callContext,
+                                           final String callSample,
+                                           final int minGq, final int minDp) {
+        return classifyVariants(truthContext, truthSample, callContext, callSample, Optional.empty(), Optional.empty(), minGq, minDp);
+    }
+
+    /**
+     * Attempts to determine the concordance state given the truth and all variant context.  Optionally increments the genotype concordance
+     * counts for SNPs and Indels respectively.  This will ignore cases where an indel was found the truth and SNP  was found  in the call,
```

Comment:
Also, the grammer is a little messed up.  Please fix.
It will only update the count for a snp or an indel but not both, correct?